kiwi/system/setup.py

```diff
@@ -87,6 +88,48 @@ def __init__(self, xml_state: XMLState, root_dir: str):
         self._preferences_lookup()
         self._oemconfig_lookup()
 
+    def setup_registry_import(self) -> None:
+        """
+        Fetch container(s) and activate systemd unit to load
+        fetched containers during boot
+        """
+        container_files_to_load = []
+        container_execs_to_load = []
+        after_services = set()
+        for container in self.xml_state.get_containers():
+            log.info(f'Fetching container: {container.name}')
+            pathlib.Path(f'{self.root_dir}/{defaults.LOCAL_CONTAINERS}').mkdir(
+                parents=True, exist_ok=True
+            )
+            Command.run(
+                ['chroot', self.root_dir] + container.fetch_command
+            )
+            if container.load_command:
+                container_files_to_load.append(container.container_file)
+                container_execs_to_load.append(container.load_command)
+                if container.backend == 'docker':
+                    after_services.add('docker.service')
+
+        if container_files_to_load:
+            log.info('--> Setup kiwi_containers.service import unit')
+            service = BuilderTemplateSystemdUnit()
+            unit_template = service.get_container_import_template(
+                container_files_to_load, container_execs_to_load,
+                list(after_services)
+            )
+            unit = unit_template.substitute()
+            unit_file = '{0}/usr/lib/systemd/system/{1}.service'.format(
```

Comment:
Why are we writing this in `/usr` instead of `/etc`? Even if we're writing into `/usr`, we should put it in `/usr/local` rather than `/usr`, since this is not packaged content.

Reply:
good catch, yes this needs to change

Reply:
fixed